src/components/blocks/DocumentInfo.tsx

```diff
@@ -21,6 +21,9 @@ type TDoucmentInfoProps = {
 };
 
 const DocumentInfo = ({ heading, text = "", list = [], id = "", tooltip = "", bulleted = false }: TDoucmentInfoProps) => {
+  const dataProps = {};
+  dataProps[`data-analytics-${heading.toLowerCase().trim().replace(" ", "-")}`] = "";
```

Comment:
Maybe worth protecting this a little more, eg having a fixed list of allowed keys somewhere to check against?